doc/user_guide/model.rst

```diff
@@ -63,7 +63,7 @@ parameters for spectroscopy than the one that ships with HyperSpy:
 
 .. code-block:: python
 
-    >>> g = hs.components.Expression(
+    >>> g = hs.model.components.Expression(
```

Comment:
Good that you raise this: I thought that, although at present ``components`` are alone in ``model``, the arrival of SAMFire in HyperSpy 0.9 will change this, making it worth it to have a ``model`` submodule.